packages/location-management/src/components/LocationUnitAddEdit/tests/index.test.tsx

```diff
@@ -13,25 +13,24 @@ import LocationUnitAddEdit, { getBaseTreeNode, getHierarchy } from '..';
 
 import { act } from 'react-dom/test-utils';
 
-jest.mock('../../../configs/env');
-
 describe('location-management/src/components/LocationUnitAddEdit', () => {
+  const baseURL = 'https://opensrp-stage.smartregister.org/opensrp/rest/';
```

Comment:
@haseemVD can we get this from constants

Reply:
you can then pass it as the default prop to the respective components and that you do not need to change all the test suits where those components are defined.

Reply:
What if I declare the const in config/env, and then use it everywhere for tests in this module.
does it make sense ?